src/test/java/stepDefinitions/StepsSignUp.java

```diff
@@ -61,7 +64,7 @@ public void iLeaveTheFormFieldsBlank() {
 
     @Then("the system displays error messages corresponding to blank fields")
     public void theSystemDisplaysErrorMessagesCorrespondingToBlankFields() {
-        Assert.assertTrue(FIELDS_BLANCK, singUpPage.getErrorMessages().containsAll(FIELDS_CAN_NOT_BLANCK));
+        Assert.assertTrue(FIELDS_BLANCK, FIELDS_CAN_NOT_BLANCK.containsAll(singUpPage.getErrorMessages()));
```

Comment:
If you are asserting that a string is equal to another one, I recommend using `equalsIgnoreCase` instead of `containsAll`

Reply:
It is not a comparison between strings, it is a comparison between lists :)